lib/widgets/compose_box.dart

```diff
@@ -903,45 +904,47 @@ class _ComposeBoxLayout extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    ThemeData themeData = Theme.of(context);
-    ColorScheme colorScheme = themeData.colorScheme;
+    final themeData = Theme.of(context);
+    final designVariables = DesignVariables.of(context);
 
     final inputThemeData = themeData.copyWith(
-      inputDecorationTheme: InputDecorationTheme(
-        // Both [contentPadding] and [isDense] combine to make the layout compact.
+      inputDecorationTheme: const InputDecorationTheme(
         isDense: true,
-        contentPadding: const EdgeInsets.symmetric(
-          horizontal: 12.0, vertical: _inputVerticalPadding),
-        border: const OutlineInputBorder(
-          borderRadius: BorderRadius.all(Radius.circular(4.0)),
-          borderSide: BorderSide.none),
-        filled: true,
-        fillColor: colorScheme.surface,
-      ),
-    );
+        // Leaving `contentPadding` out would still result in the same layout.
+        // For better predictability, set it explicitly instead of relying on
+        // the default value.
+        contentPadding: EdgeInsets.zero,
```

Comment:
This didn't turn out to be true in my testing just now:

| with `EdgeInsets.zero` | left out |
| --- | --- |
| <img width="486" alt="image" src="https://github.com/user-attachments/assets/9d2e2f1a-b0b9-4ce9-9d3e-a1f0133cd789"> | <img width="486" alt="image" src="https://github.com/user-attachments/assets/895f88af-094c-45cd-9cbf-1bb519a8b6c6"> |

I see the layout shifting if I open these images in separate browser tabs and switch between them.